DuckDuckGo/da.lproj/Localizable.strings

```diff
@@ -1080,25 +1077,31 @@
 "duckPlayer.never.label" = "Aldrig";
 
 /* Body text for the modal feature explanation */
-"duckplayer.presentation.modal.body" = "Duck Player giver dig mulighed for at se YouTube uden målrettede annoncer i en biograflignende oplevelse i DuckDuckGo, og det, du ser, påvirker ikke dine anbefalinger.";
+"duckplayer.presentation.modal.body" = "Duck Player lets you watch YouTube without targeted ads in DuckDuckGo and what you watch won’t influence your recommendations.";
```

Comment:
This change seems incorrect?

Reply:
This was one of two excluded `duckplayer` strings and apparently it also changed the value. Removed exclusion and re-imported the strings, should be fine now.